torchtext/models/roberta/bundler.py

```diff
@@ -156,10 +155,13 @@ def encoderConf(self) -> RobertaEncoderConf:
 XLMR_BASE_ENCODER = RobertaModelBundle(
     _path=urljoin(_TEXT_BUCKET, "xlmr.base.encoder.pt"),
     _encoder_conf=RobertaEncoderConf(vocab_size=250002),
-    transform=partial(get_xlmr_transform,
-                      vocab_path=urljoin(_TEXT_BUCKET, "xlmr.vocab.pt"),
-                      spm_model_path=urljoin(_TEXT_BUCKET, "xlmr.sentencepiece.bpe.model"),
-                      )
+    transform=lambda: T.Sequential(
```

Comment:
Assigning lambda is an anti-pattern.

https://docs.quantifiedcode.com/python-anti-patterns/correctness/assigning_a_lambda_to_a_variable.html

Reply:
hmm, reading this, it might not

https://stackoverflow.com/a/27928036

Reply:
yet I feel overwhelmed by the fact that this lambda spans multiple lines.

Reply:
hmm, interesting I wasn't aware of this around lambda usage. thanks for sharing. I honestly thought it looks nicer being able to define a lazy transform in-place :). The alternative would be to define a private function that returns the composite transform.

Reply:
yeah, I am looking at all the possible reasoning, but I am also being convinced that `lambda` is the simplest here.

Reply:
Another way would be to compose a no-op lambda with the rest of the transform with `partial`, though it isn't as easy to read.